### PR TITLE
#163205033 Modified blueprints and test file

### PR DESCRIPTION
**Pull request by samwanjo41**
**What does this PR do ?**
- Modified blueprints
- Modified test file for meetups

**Description of tasks to be done**
-Modified blueprints to make sure blueprint decorators are uniform across all files
-   The test_meetups file was updated to remove json dumps on get methods
- Added asserts for messages

**How should you manually test this?**
Clone the repo git clone https://github.com/samwanjo41/Questioner.git
cd into the project folder
Activate virtual env: source/bin/activate
Use POSTMAN to test the API endpoints
Navigate to app/tests/v1
On the terminal, run pytest

**What are the relevant PT stories?**
#163205033

### CODE REVIEW

File: app/api/v1/views/meetups_view.py [hunk 1]
```diff
@@ -33,27 +33,40 @@ def createMeetup():
...
+    if meetups:
+        return make_response(jsonify({
         "status": 200,
         "data": meetups
     }), 200)
+    else:
+        return make_response(jsonify({
+        "status": 404,
+        "error": "No meetups created yet"
+    }), 400)
```
Comment (dessHub) [inline app/api/v1/views/meetups_view.py]: Status code should be consistent.

File: app/tests/v1/test_meetups.py [hunk 2]
```diff
@@ -24,28 +25,43 @@ def setUp(self):
...
+       
 
     def test_specific_meetups(self):
         """user view specific meetup by id"""
-        response = self.client.get('/api/v1/meetups/1', data=json.dumps(self.meetup), content_type='application/json')
+        today = datetime.utcnow().isoformat()
+        response = self.client.get('/meetups/1', content_type='application/json')
+        result = json.loads(response.data.decode('utf-8'))
         self.assertEqual(response.status_code, 200)
+        self.assertEqual(result["status"], 200)
```
Comment (dessHub) [inline app/tests/v1/test_meetups.py]: Add assertion for the data.

**Comment by dessHub** [comment]
Good work, just take care of the failing tests.

**Review by samwanjo41 — COMMENTED** [review]
changed meetups view file error

**Pull request merged**